src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -713,7 +713,7 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
 
     renderTriggerModalCTA() {
         const buttonLabel = CDButtonLabelMap[this.props.stageType]
-
+        const cdModalHeader = ['Pre Deployment' , 'Post Deployent']
```

Comment:
Spelling for deployment is wrong, and you said you tested it?

Reply:
done